hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconStorageContainerManagerFacade.java

```diff
@@ -383,32 +420,41 @@ private OzoneConfiguration getReconScmConfiguration(
    * Start the Recon SCM subsystems.
    */
   @Override
-  public void start() {
+  public void start() throws IOException {
     if (LOG.isInfoEnabled()) {
       LOG.info(buildRpcServerStartMessage(
           "Recon ScmDatanodeProtocol RPC server",
           getDatanodeProtocolServer().getDatanodeRpcAddress()));
     }
+    reconSCMSyncScheduler = Executors.newScheduledThreadPool(1, threadFactory);
+    registerSCMDBTasks();
+    try {
+      scmMetadataManager.start(ozoneConfiguration);
+    } catch (IOException ioEx) {
+      LOG.error("Error staring Recon SCM Metadata Manager.", ioEx);
+    }
+    reconTaskController.start();
+    long initialDelay = ozoneConfiguration.getTimeDuration(OZONE_RECON_SCM_SNAPSHOT_TASK_INITIAL_DELAY,
+            OZONE_RECON_SCM_SNAPSHOT_TASK_INITIAL_DELAY_DEFAULT, TimeUnit.MILLISECONDS);
+
+    // This schedules a periodic task to sync Recon's copy of SCM metadata
+    // with SCM metadata rocks DB. Gets full snapshot or delta updates.
```

Comment:
Do we need PipelineSyncTask anymore? I think we can remove it as we are getting the entire db updated.

Reply:
I do not want to remove this task because this task queries pipelines and their state from SCM which are not yet flushed to SCM DB and I still see ratis transaction threshold as 10000 , crossing which it forces to flush in DB. Also I am not sure why but see this task is also having a call for syncing operational state on dead nodes.